services/src/handlers/ebv.rs

```diff
@@ -1464,4 +1545,98 @@ mod tests {
             }
         );
     }
+
+    #[tokio::test]
+    async fn test_remove_overviews() {
+        crate::util::config::set_config(
+            "session.admin_session_token",
+            "8aca8875-425a-4ef1-8ee6-cdfc62dd7525",
+        )
+        .unwrap();
+
+        let ctx = InMemoryContext::test_default();
+        let admin_session_id = AdminSession::default().id();
+
+        let overview_folder = tempfile::tempdir().unwrap();
+
+        ctx.layer_provider_db_ref()
+            .add_layer_provider(Box::new(NetCdfCfDataProviderDefinition {
+                name: "test".to_string(),
+                path: test_data!("netcdf4d").to_path_buf(),
+                overviews: overview_folder.path().to_path_buf(),
+            }))
+            .await
+            .unwrap();
+
+        let req = actix_web::test::TestRequest::post()
+            .uri("/ebv/createOverview")
+            .append_header((
+                header::AUTHORIZATION,
+                Bearer::new(admin_session_id.to_string()),
+            ))
+            .set_json(serde_json::json!({
+                "file": "dataset_m.nc"
+            }));
+
+        let res = send_test_request(req, ctx.clone(), "http://test".to_string()).await;
+
+        assert_eq!(res.status(), 200, "{:?}", res.response());
+
+        let task_response =
+            serde_json::from_str::<TaskResponse>(&read_body_string(res).await).unwrap();
+
+        wait_for_task_to_finish(ctx.tasks(), task_response.task_id).await;
+
+        let status = ctx.tasks().status(task_response.task_id).await.unwrap();
+
+        let mut response = if let TaskStatus::Completed { info } = status {
+            info.as_any()
+                .downcast_ref::<NetCdfCfOverviewResponse>()
+                .unwrap()
+                .clone()
+        } else {
+            panic!("Task must be completed");
+        };
+
+        response.success.sort();
+        assert_eq!(
+            response,
+            NetCdfCfOverviewResponse {
+                success: vec!["dataset_m.nc".into()],
+                skip: vec![],
+                error: vec![],
+            }
+        );
+
+        // Now, delete the overviews
+
+        let req = actix_web::test::TestRequest::post()
+            .uri("/ebv/removeOverview")
+            .append_header((
+                header::AUTHORIZATION,
+                Bearer::new(admin_session_id.to_string()),
+            ))
+            .set_json(serde_json::json!({
+                "file": "dataset_m.nc"
+            }));
+
+        let res = send_test_request(req, ctx.clone(), "http://test".to_string()).await;
+
+        assert_eq!(res.status(), 200, "{:?}", res.response());
+
+        let task_response =
+            serde_json::from_str::<TaskResponse>(&read_body_string(res).await).unwrap();
+
+        wait_for_task_to_finish(ctx.tasks(), task_response.task_id).await;
+
+        let status = ctx.tasks().status(task_response.task_id).await.unwrap();
+
+        assert_eq!(
```

Comment:
maybe also check if the files were actually deleted

Reply:
[bd83581](https://github.com/geo-engine/geoengine/pull/592/commits/bd835810a808f93fa3666b9d715a6b03f9c3ea6f)